python/sglang/srt/model_executor/model_runner.py

```diff
@@ -220,6 +223,9 @@ def initialize(self, min_per_gpu_memory: float):
             self.init_cublas()
             self.init_attention_backend()
             self.init_cuda_graphs()
+        elif self.device == "hpu":
+            self.init_attention_backend()
+            self.cuda_graph_runner = HPUGraphRunner(self)
```

Comment:
It's a bit confusing to assgin HPUGraphRunner to cuda_graph_runner
Maybe seperate to self.hpu_graph_runner from cuda_graph_runner and add self.init_hpu_graphs() as self.init_cuda_graphs()

Reply:
I have thought about changing the name to something more generic like just "graph_runner", but there are many "cuda_graph" references in the current code, so I would prefer to the refactor to future PR to limit to size of this PR.